flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -257,20 +257,25 @@
 		key("taskmanager.memory.process.size")
 			.memoryType()
 			.noDefaultValue()
-			.withDescription("Total Process Memory size for the TaskExecutors. This includes all the memory that a"
-				+ " TaskExecutor consumes, consisting of Total Flink Memory, JVM Metaspace, and JVM Overhead. On"
-				+ " containerized setups, this should be set to the container memory.");
+			.withDescription("Total Process Memory size for the TaskExecutors. This includes all the memory that a "
+				+ "TaskExecutor consumes, consisting of Total Flink Memory (configured with "
+				+ "'taskmanager.memory.flink.size'), JVM Metaspace, and JVM Overhead. On "
+				+ "containerized setups, this should be set to the container memory."
```

Comment:
Adding the parenthesis with the reference to `flink.size` somehow reads as if one needs to configure this in order to make `process.size` work. I would rather suggest to add a sentence `See also taskmanager.memory.flink.size for flink process mmeory size configuration`.